responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/review-support/review-support.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/internal/Observable';
+import { Support } from 'src/app/core/api/models/support';
+import { RegistrationsService } from 'src/app/core/api/services';
+
+@Injectable({ providedIn: 'root' })
+export class ReviewSupportService {
+  private includeEvacueeSummaryVal: boolean;
+
+  constructor(private registrationService: RegistrationsService) {}
+
+  public get includeEvacueeSummary(): boolean {
```

Comment:
This should change once the Support model updates and directly set in the support model. There is no need for storing this separately since this value can only be set once and is never referenced again